rsciio/image/_api.py

```diff
@@ -167,7 +168,15 @@ def file_writer(
         # Sanity check of the axes
         # This plugin doesn't support non-uniform axes, we don't need to check
         # if the axes have a scale attribute
-        if axes[0]["scale"] != axes[1]["scale"] or axes[0]["units"] != axes[1]["units"]:
+        if (
+            axes[0]["units"] != axes[1]["units"]
+            or
+            # relative difference normalized to the size of the second axis
+            # the more pixels is in the second axis, the higher the tolerance need to be
```

Comment:
```suggestion
            # the more pixels are in the second axis, the higher the tolerance needs to be
```

I am not sure if "higher tolerance" is the right term here? Your tolerance value gets smaller for axes with more pixels.